cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/slurmdbd.service.erb

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Slurm DBD accounting daemon
+After=network-online.target munge.service mysql.service mysqld.service mariadb.service
+Wants=network-online.target
+ConditionPathExists=<%= node['cluster']['slurm']['install_dir'] %>/etc/slurmdbd.conf
+
+[Service]
+Type=simple
+EnvironmentFile=-/etc/sysconfig/slurmdbd
+ExecStart=<%= node['cluster']['slurm']['install_dir'] %>/sbin/slurmdbd -D -s $SLURMDBD_OPTIONS
+ExecReload=/bin/kill -HUP $MAINPID
+LimitNOFILE=65536
+TasksMax=infinity
```

Comment:
I see this is not included in the service file generated at compile time. Can you leave a comment on why this is necessary for future reference?

Reply:
Do you mean the `TasksMax` parameter? Or the whole block?

If you are talking about `TasksMax`, this is actually coming from [this commit](https://github.com/SchedMD/slurm/commit/02aa5a1a20cd97a238801e99ecb7af1df0022e39) in Slurm. It looks like it is being added automatically if the installer (Makefile) sees a version of systemd >= 227. Actually this is done for all the Slurm services.

I'll remove this and add it for all services once we check that we have a suitable version of systemd on all our AMIs.

Reply:
Done.